R-Interface/jasprcpp.cpp

```diff
@@ -412,12 +425,14 @@ const char*	STDCALL jaspRCPP_evalRCode(const char *rCode) {
 	rinside->instance()[".rCode"] = CSTRING_TO_R(rCode);
 	const std::string rCodeTryCatch(""
 		"returnVal = 'null';	"
-		"tryCatch(				"
-		"	suppressWarnings({	returnVal <- eval(parse(text=.rCode))     }),	"
-		"		error	= function(e) { .setRError(toString(e$message))	  }, 	"
-		"		warning	= function(w) { .setRWarning(toString(w$message)) }		"
-		");			"
-		"returnVal	");
+	//	"tryCatch("
+		"  withCallingHandlers(	"
```

Comment:
@vandenman does it makes sense what I did here?

Reply:
well, a little bit, assuming it's necessary to print the error message as it is thrown in R.